src/Components/Health/Checker/PerformanceChecker/AbstractCompressionChecker.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Frosh\Tools\Components\Health\Checker\PerformanceChecker;
+
+use Frosh\Tools\Components\Health\Checker\CheckerInterface;
+use Frosh\Tools\Components\Health\HealthCollection;
+use Frosh\Tools\Components\Health\SettingsResult;
+
+abstract class AbstractCompressionChecker implements PerformanceCheckerInterface, CheckerInterface
```

Comment:
wdym about doing it inside one class instead of 3. kinda just call a private method two times from `collect`

Reply:
Done